src/DevSoc.Eventathon/ClientApp/src/api/events-endpoint.js

```diff
@@ -35,6 +35,14 @@ export const useUser = () => {
     return useSWR(`/api/users/current`, fetcher);
 }
 
+export const SendAttendance = (givenID) => {
```

Comment:
I've split these out in the PR I just merged. Make sure you move this to the correct place